### PR TITLE
[Runner -> Docker driver] #60 issue handling on docker container

### PR DESCRIPTION
**Pull request by SuhoSon**
Originally, docker driver pulls the image from docker hub for create a container.
But, this approach cannot apply local changes for trace_replay,
I create the local version new image included trace-replay binary changed compiled by user for apply changes and use it.

### CODE REVIEW

**Comment by SuhoSon** [comment]
Additionally, solve the deadlock issue from test program

File: runner/test/docker-driver-test.c [hunk 1]
```diff
@@ -169,8 +169,8 @@ void test(void)
                 for (unsigned long i = 0; i < key_len; i++) {
                         struct json_object *object, *tmp;
                         int type;
-                        if (flags == FLAGS_MASK) {
-                                break;
+                        if (flags & (0x1 << i)) {
+                                continue;
```
Comment (BlaCkinkGJ) [inline runner/test/docker-driver-test.c:173]: `FLAGS_MASK` macro value is changed. For this reason, you don't have to change this lines.

**Review by BlaCkinkGJ — CHANGES_REQUESTED** [review]
Please check following comments.....

File: runner/test/docker-driver-test.c (hunk 1 above)
Comment (SuhoSon) [reply runner/test/docker-driver-test.c:173]: But already check the FLAGS_MASK outer.
This change just to skip the already finished cgroup.

Comment (BlaCkinkGJ) [reply runner/test/docker-driver-test.c:173]: Well, I agree with that opinion. I will squash this PR.

**Review by BlaCkinkGJ — APPROVED** [review]
Confirm

**Pull request merged**